app/Http/Controllers/Auth/AuthController.php

```diff
@@ -11,115 +13,155 @@
-		// to login and redirect the user back to the login form. Of course, when this
-		// user surpasses their maximum number of attempts they will get locked out.
-		if ($throttles) {
-			$this->incrementLoginAttempts ( $request );
-		}
-		
-		return redirect ( $this->loginPath () )->withInput ( $request->only ( $this->loginUsername (), 'remember' ) )->withErrors ( [ 
-				$this->loginUsername () => $this->getFailedLoginMessage () 
-		] );
-	}
-	
-	/**
-	 * Handle a registration request for the application.
-	 *
-	 * @param \Illuminate\Http\Request $request        	
-	 * @return \Illuminate\Http\Response
-	 */
-	public function postRegister(Request $request) {
-		$validator = $this->validator ( $request->all () );
-		
-		if ($validator->fails ()) {
-			return redirect ( $this->registerPath )->withErrors ( $validator );
-		}
-		
-		Auth::login ( $this->create ( $request->all () ) );
-		return redirect ( $this->redirectPath () );
-	}
-	
-	/**
-	 * Get a validator for an incoming registration request.
-	 *
-	 * @param array $data        	
-	 * @return \Illuminate\Contracts\Validation\Validator
-	 */
-	protected function validator(array $data) {
-		return Validator::make ( $data, [ 
-				'name' => 'required|max:255|unique:users,username|min:3',
-				'email' => 'required|email|max:255|unique:users',
-				'password' => 'required|confirmed|min:8' 
-		] );
-	}
-	
-	/**
-	 * Create a new user instance after a valid registration.
-	 *
-	 * @param array $data        	
-	 * @return User
-	 */
-	protected function create(array $data) {
-		return User::create([
-                'email' => $data['email'],
-            		  'username' => $data['name'],
-                'password' => bcrypt($data['password']),
-            		'status' =>TRUE,
-            		'profile_state' => FALSE
-            ]);
-        }
+  /*
+   * |--------------------------------------------------------------------------
+   * | Registration & Login Controller
+   * |--------------------------------------------------------------------------
+   * |
+   * | This controller handles the registration of new users, as well as the
+   * | authentication of existing users. By default, this controller uses
+   * | a simple trait to add these behaviors. Why don't you explore it?
+   * |
+   */
+  
+  use AuthenticatesAndRegistersUsers, ThrottlesLogins;
+  protected $loginPath = '/login';
+  protected $registerPath = '/register';
+  protected $redirectPath = '/';
+  /**
+   * Create a new authentication controller instance.
+   *
+   * @return void
+   */
+  public function __construct() {
+    $this->middleware ( 'guest', [ 
+        'except' => 'getLogout' 
+    ] );
+  }
+  public function socialLogin(SocialAuthenticateUser $authenticateUser, Request $request, $provider = null) {
+    $social_providders = array (
+        "facebook",
+        "google" 
+    );
+    if (in_array ( strtolower ( $provider ), $social_providders )) {
```

Comment:
Consistency is key, Some functions you wrote had no space, this and some others have space within. Please, be consistent. Fix!